app/helpers/compute_resources_vms_helper.rb

```diff
@@ -294,4 +294,10 @@ def vm_console_action(vm)
       }
     )
   end
+
+  def vmware_vm_hypervisor_name(vm)
+    vm.hypervisor&.name
+  rescue
+    nil
```

Comment:
Would it be more useful for the users instead of showing an empty field if we give more information, e.g.:
```suggestion
  rescue NoPermissions
    'Missing Permissions'
  rescue StandardError => e
    "Error: #{e}"
```

Reply:
If we catch the `StandardError` it might be a good idea to log this: `Foreman::Logging.exception('Caught an error when retrieving hypervisor information.', e)`